pytraj/externals/_json.py

```diff
@@ -43,5 +43,5 @@ def try_read(path):
             with open(path, mode) as fh:
                 return json.load(fh)
         except (Exception) as e:
-            raise
+            raise e
```

Comment:
pyflakes will still complain about `e` being unused (which you can get rid of by removing `as e`, since you don't need the exception).

But this is curious because

```Python
try:
    do_something()
except Exception:
    raise
```

is exactly equivalent to

```python
do_something()
```